playwright.config.ts

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
   /* Run tests in files in parallel */
   fullyParallel: true,
   timeout: 250000,
-  expect: { timeout: 250000 },
+  expect: { timeout: 90000 },
```

Comment:
is there a reason to change/reduce the timeout?

Reply:
Issue updated